src/libs/ajax/AzureStorage.ts

```diff
@@ -165,7 +170,8 @@ export const AzureStorage = (signal?: AbortSignal) => ({
     };
 
     return {
-      getData: getObjectMetadata,
+      getMetadata: () => getBlobByUri('HEAD'),
+      getData: () => getBlobByUri('GET'),
```

Comment:
Do you need to mark these as async too?

Reply:
this occurred to me too - I figured that a successful manual test meant that I don't, but I'll double check before merging